CHANGELOG.rst

```diff
@@ -1,42 +1,54 @@
 3.0.0.dev1
 ==========
 
-**Changes**
+**Incompatible changes**
 
-*
+* Removing the following deprecated commandline options
```

Comment:
Hmm I noticed we lack consistency in all messages actually: Fixed, Fix, ... not sure if you want to review all entries though.